org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/text/correction/QuickAssistProcessor.java

```diff
@@ -233,7 +224,8 @@ public List<ProposalKindWrapper> getAssists(CodeActionParams params, IInvocation
 				//				getMissingCaseStatementProposals(context, coveringNode, resultingCollections);
 			getStringConcatToTextBlockProposal(context, coveringNode, resultingCollections);
 			// }
-			getAddMethodDeclaration(context, coveringNode, resultingCollections);
+			//getAddMethodDeclaration(context, coveringNode, resultingCollections);

```

Comment:
This commented out line can be removed right ?

Reply:
Yes i will push the changes during today.